### DISCUSSION TITLE
[Builder Edition] Problems with IceWM to LXQt

### DISCUSSION THREAD

**Discussion by birds-swim** [SpiralLinux/SpiralLinux-project#255] [Q&A] [answered]
Edit: I will provide screenshots later. I don't have the time to do that right now as I am at work.

Correction:  When I say "missing/no icons" I really mean they have icons, but they look like weird placeholders for a later icon pack or something. Installing an icon pack from the Debian repos *does not* fix this problem.  
____________________________________________________
I noticed when I try to go from IceWM to LXQt in the SpiralLinux Builder Edition, some of the icons are missing in LXQt. Icons in the panel and in the start menu and the LXQt Settings (the 3 window management options have placeholder icons). 

After installing LXQt, I always delete IceWM and install KWin or Xfwm (used by Debian's LXQt version). If you don't, then LXQt and IceWM fight each other and your LXQt panel will look weird.

I was told by some friends in the LXQT matrix chat that LXQT inherits "some stuff" from the previous session or DE. Not exactly sure what they're talking about, but they must mean there are some configs in my `/home` directory that LXQT relies on or borrows for its first initialization and setup.

I don't know where these configs are; I don't know where to look for them. My question today is this: Where would the files that IceWM and LXQt use be located? Specifically which folder? If I can learn this, then this will greatly help me in my Linux Journey. 

I have tested a Debian Netinstall and going from console to LXQt, and then I'm presented with all the icons I need.

**Comment by geckolinux** [comment, marked as answer]
Hi there, the Builder edition has almost no customization at all to Icewm. I suspect that you're missing some packages for LXQt and/or you're not using the correct LXQt config module. LXQt is an extremely chaotic mix of many little disjointed elements, some of which have duplicate functionality or conflict with each other. I would strongly recommend that you just use the SpiralLinux LXQt edition, which has already worked around most of those oddities. Or if you want to learn which config files are applicable then you can download the `tar.gz` from Github and inspect the `/config/includes.chroot/etc/skel/` directory (with hidden dotfiles visible).

**Reply by birds-swim** [reply]
I really like Lubuntu's look. Will it be feasible to fire up a VM and steal their configs for my setup? After examining the tar archive, I think that would be very helpful.

If all else fails, then I'll just use the Spiral one.

I'm going out of my way here on purpose: I'm trying chase after the r/unixporn leaderboards using a pure Qt setup.

Thanks for your response, Dev! It was very helpful.
